storey/targets.py

```diff
@@ -902,16 +934,26 @@ async def _emit(self, batch, batch_key, batch_time, batch_events, last_event_tim
                 b.write(self._table)
             else:  # table is dynamic
                 b.write(batch_key)
+            if self._supertable:
+                b.write(" USING ")
+                if not self._using_websocket:
+                    b.write(self._database)
+                    b.write(".")
+                b.write(self._supertable)
+            if self._number_of_tags:
```

Comment:
this `if` is redundant because you already validated that if you have a supertable, you must have tags.  
```suggestion   
```